bench_rpc/src/bench.cpp

```diff
@@ -726,15 +727,9 @@ void bench_run_all(void){
 
     //Echo application, capture RTT latency and throughput
     //TODO: Modify to pass arguments
-    single_rtt_test();
-    single_connection_test();
-
-#ifdef __APPLE__
```

Comment:
@EIRNf : wouldn't this be better to just have the bench tests run up to the limits of osx, rather than not run at all?